### PR TITLE
Upd bug template to use label 'bug'; ask for flux logs

### PR DESCRIPTION
**Pull request by palemtnrider**
**What changed?**
Update bug issue template to use label `bug`

**Why?**
Auto-assign the bug label when someone chooses that template.

**How did you test it?**

**Release notes**
N/a

**Documentation Changes**
n/a

### CODE REVIEW

**Review by J-Thompson12 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'type/bug'
+labels: 'bug'
 assignees: ''
 
 ---
```
Comment (rokshana-b) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]: @palemtnrider Can you add a comment below in the template that prompts or at least reminds the issuer to mark their bug's severity level? Something like:

**Describe the bug**
A clear and concise description of what the bug is. Please choose a severity label for your bug.

Not sure if my suggestion holds any validity. It was just a passing thought :)

Comment (palemtnrider) [reply .github/ISSUE_TEMPLATE/bug_report.md:8]: I'm happy to add it, do we have a doc I can point to that provides a bit of guidance to the reporter?  i.e., a markdown file in the repo? (I didn't see one). If you prefer we could add the information to the template.  @rokshana-b

Comment (rokshana-b) [reply .github/ISSUE_TEMPLATE/bug_report.md:8]: @palemtnrider There's no markdown file, but there is a google doc with guidelines: https://docs.google.com/document/d/18K9WpnMwIctIXWNLuqHqCvGhCww63jZuC6q3pJBhhz4/edit

Comment (palemtnrider) [reply .github/ISSUE_TEMPLATE/bug_report.md:8]: @rokshana-b I've added a prompt for severity.  wdyt?

Comment (rokshana-b) [reply .github/ISSUE_TEMPLATE/bug_report.md:8]: @palemtnrider tysm, this is perfect!

**Review by rokshana-b — APPROVED** [review]

**Review by JamWils — APPROVED** [review]
LGTM

**Pull request merged**